chef/automate
# Add CreatedAt field to the GetNodes response

![image](https://user-images.githubusercontent.com/1679247/80432836-326c5600-88aa-11ea-8bfd-84e47a6afbc9.png)

### :nut_and_bolt: Description: What code changed, and why?

Adding the CreatedAt field to the GetNodes response

### :athletic_shoe: How to Build and Test the Change
1. `build components/config-mgmt-service && start_all_services`
1. Add some nodes with `chef_load_nodes 10`
1. Go to https://a2-dev.test/infrastructure/client-runs and check the response for the "https://a2-dev.test/api/v0/cfgmgmt/nodes" request. It should look like the above image. 

### :white_check_mark: Checklist
